spec/requirements_spec.rb

```diff
@@ -0,0 +1,183 @@
+require 'rails_helper'
+
+RSpec.describe 'Requirements with without user logged in', type: :system do
+  context 'splash page' do
+    it 'should diplay a splash page if user not logged in' do
+      visit groups_path
+      expect(page).to have_selector('.splash')
+    end
+    it 'should display a link to the login page if user not logged in' do
+      visit groups_path
+      expect(page).to have_link('LOG IN')
+    end
```

Comment:
- [ ] You have done well writing tests for this project 👏 However, this test is failing. Kindly fix it so it passes 👍
![image](https://user-images.githubusercontent.com/46839548/182972155-63889f04-be9e-476e-bb42-25268efd4bba.png)

Reply:
Will do 👍